projects/DensePose/doc/GETTING_STARTED.md

```diff
@@ -9,6 +9,11 @@ python apply_net.py show configs/densepose_rcnn_R_50_FPN_s1x.yaml densepose_rcnn
 ```
 Please see [Apply Net](TOOL_APPLY_NET.md) for more details on the tool.
 
+Alternatively, you may choose to consume the models as an API from the following site(s):
```

Comment:
`Alternatively, you may choose to consume the models as an API from the following site(s)` -> This line seems like an endorsement.  Reviewers may not accept this

Reply:
When you create PR, changes in the below 2 files should be good enough:
```
./README.md
./projects/DensePose/README.md
```